cpmpy/expressions/globalfunctions.py

```diff
@@ -126,20 +142,15 @@ def value(self):
         else:
             return min(argvals)
 
-    def decompose_comparison(self, cpm_op, cpm_rhs):
+    def decompose(self):
         """
-        Decomposition if it's part of a comparison
-
-        Returns two lists of constraints:
-
-        1) constraints representing the comparison
-        2) constraints that (totally) define new auxiliary variables needed in the decomposition,
-           they should be enforced toplevel.
+        Decomposition of Minimum constraint.
+        Returns
```

Comment:
this is a generic comment, ideally this is only needed in the abstract class?

Here it could be something more specific that actually explains this not entirely trivial decomp, like
"Can only be decomposed by introducing an auxiliary variable and constraining it is smaller than each variable, while at the same time not being smaller then all (e.g. it needs to be (larger or) equal to one of them)"